CMakeLists.txt

```diff
@@ -348,14 +347,16 @@ if(EMSCRIPTEN)
   if(BUILD_FOR_BROWSER)
     add_link_flag("-sENVIRONMENT=web,worker")
     add_link_flag("-sINVOKE_RUN=0")
-    add_link_flag("-sEXPORTED_RUNTIME_METHODS=run,callMain,FS")    
+    add_link_flag("-sEXPORTED_RUNTIME_METHODS=run,callMain,FS")
     add_link_flag("-sMODULARIZE")
     add_link_flag("-sEXPORT_ES6")
     add_link_flag("-sFILESYSTEM")
     add_link_flag("-sFORCE_FILESYSTEM")
+    add_link_flag("-sSTACK_SIZE=5242880")
```

Comment:
I this specific to the browser? If not let's put it up so it affects node.js as well?

Also this can be `-sSTACK_SIZE=5MB`

Reply:
Moved up and changed the unit.